### PR TITLE
Changes: in AmazonOrderList to create the multiple marketplaces request

### PR DESCRIPTION
**Pull request by chaoukibenfradj**
Have the ability to create an OrderList request to get orders for multiple Marketplaces at the same time.
The changes that I made, only concerns the AmazonOrderList class. Other services from the SDK are not affected with the changes.

### CODE REVIEW

File: tests/mocks/classes/Config.php [hunk 1]
```diff
@@ -7,6 +7,12 @@
 
 class Config
 {
+    const MARKETPLACE_1 = 'A1PA6795UKMFR9';
+    const MARKETPLACE_2 = 'APJ6JRA9NG5V4';
+    const MARKETPLACE_3 = 'A13V1IB3VIYZZH';
+    const MARKETPLACE_4 = 'A1RKKUPIHCS9HS';
+    const MARKETPLACE_5 = 'A1F83G8C2ARO7P';
```
Comment (PatrickBidx) [inline tests/mocks/classes/Config.php]: Please add the proper market place name to the constant:
```
const MARKETPLACE_DE = 'A1PA6795UKMFR9';
const MARKETPLACE_IT = 'APJ6JRA9NG5V4';
const MARKETPLACE_FR = 'A13V1IB3VIYZZH';
const MARKETPLACE_ES = 'A1RKKUPIHCS9HS';
const MARKETPLACE_GB = 'A1F83G8C2ARO7P';
```

File: src/AmazonOrderList.php [hunk 2]
```diff
@@ -67,7 +67,19 @@ public function __construct($s, $mock = false, $m = null)
         }
 
         if (isset($store[$s]) && array_key_exists('marketplaceId', $store[$s])) {
-            $this->options['MarketplaceId.Id.1'] = $store[$s]['marketplaceId'];
+            //Make possible to pass multiple MarketplaceIds
+            //The request will have a list of Marketplaces as:
+            //MarketplaceId.Id.x1 MarketplaceId.Id.x2 MarketplaceId.Id.x3
+            if (is_array($store[$s]['marketplaceId'])) {
+                $count_marketplace = count($store[$s]['marketplaceId']);
```
Comment (PatrickBidx) [inline src/AmazonOrderList.php]: Please use CamelCase for `$count_marketplace`.

File: src/AmazonOrderList.php [hunk 3]
```diff
@@ -67,7 +67,19 @@ public function __construct($s, $mock = false, $m = null)
...
+                for ($i = 0; $i < $count_marketplace; ++$i) {
+                    $parameter_increment = $i + 1;
```
Comment (PatrickBidx) [inline src/AmazonOrderList.php]: As above, please use CamelCase.

Comment (PatrickBidx) [inline src/AmazonOrderList.php]: You can just use a `foreach` loop here and initialise the `$parameter_increment` before the foreach loop with 1.

**Review by PatrickBidx — CHANGES_REQUESTED** [review]

File: src/AmazonOrderList.php (hunk 3 above)
Comment (chaoukibenfradj) [reply src/AmazonOrderList.php]: Done

File: tests/mocks/classes/Config.php (hunk 1 above)
Comment (chaoukibenfradj) [reply tests/mocks/classes/Config.php]: Done ✔️

File: src/AmazonOrderList.php (hunk 3 above)
Comment (chaoukibenfradj) [reply src/AmazonOrderList.php]: Done ✔️

File: src/AmazonOrderList.php (hunk 2 above)
Comment (chaoukibenfradj) [reply src/AmazonOrderList.php]: Done ✔️

**Review by PatrickBidx — APPROVED** [review]

**Pull request merged**